django/db/models/indexes.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import unicode_literals
+
+import hashlib
+
+from django.core.exceptions import ValidationError
+from django.utils.encoding import force_bytes
+
+__all__ = ['Index']
+
+# The max length of the names of the indexes (restricted to 30 due to Oracle)
+MAX_NAME_LENGTH = 30
+
+
+class Index(object):
+
+    index_type = 'idx'
+
+    def __init__(self, *fields, **kwargs):
```

Comment:
I don't think that just because you might add more kwargs later doesn't mean you should use kwargs instead of declaring the options you have now. Am I missing the problem?